package.json

```diff
@@ -14,9 +14,20 @@
   "license": "MIT",
   "dependencies": {
     "axios": "^0.18.0",
+    "enzyme": "^3.3.0",
```

Comment:
`enzyme` is a devDependency.

Reply:
ESLint throws an error when "enzyme" is mentioned as a devDependency, that's why made it a dependency!